pkgs/applications/editors/emacs/generic.nix

```
@@ -74,7 +72,11 @@
 , WebKit
 
 # Boolean flags
-, nativeComp ? true
+, nativeComp ? null
+, withNativeCompilation ?
+  if nativeComp != null
+  then lib.warn "nativeComp option is deprecated and will be removed in the future; use withNativeCompilation instead"
```

I don't think that works. `warn` takes two args.

```suggestion
  then lib.warn "nativeComp option is deprecated and will be removed in the future; use withNativeCompilation instead" nativeComp
```